astropy/coordinates/solar_system.py

```diff
@@ -0,0 +1,204 @@
+    if planet_index < 3:
+        barycenter_to_planet_ind = 100*planet_index + 99
+        cartesian_position_planet = (kernel[0, planet_index].compute(time.jd) +
+                                     kernel[planet_index,
+                                            barycenter_to_planet_ind].compute(time.jd))
+    # planet_index is 3 gets the moon
+    elif planet_index == 3:
+        barycenter_to_planet_ind = 301
+        cartesian_position_planet = (kernel[0, planet_index].compute(time.jd) +
+                                     kernel[planet_index,
+                                            barycenter_to_planet_ind].compute(time.jd))
+    else:
+        cartesian_position_planet = kernel[0, planet_index].compute(time.jd)
+
+    cartesian_position_earth = (kernel[0, 3].compute(time.jd) +
+                                kernel[3, 399].compute(time.jd))
+
+    # Quadrature sum of the difference of the position vectors
+    earth_distance = np.sqrt(np.sum((np.array(cartesian_position_planet) -
+                                     np.array(cartesian_position_earth))**2))*u.km
+
+    earth_to_planet_vector = u.Quantity(cartesian_position_planet -
+                                        cartesian_position_earth,
+                                        unit=u.km)
+
+    return earth_to_planet_vector, earth_distance
+
+
+def _get_apparent_planet_position(time, planet_index):
+    """
+    Calculate the apparent position of planet ``planet_index`` in cartesian
+    coordinates, given the approximate light travel time to the object.
+
+    Uses ``jplephem`` with the DE430 kernel.
+
+    Parameters
+    ----------
+    time : `~astropy.time.Time`
+        Time of observation
+
+    planet_index : int
+        Index of the planet (1-9 for Mercury through Pluto)
+
+    Returns
+    -------
+    cartesian_position : `~astropy.coordinates.CartesianRepresentation`
+        Position of the planet defined as a vector from the Earth.
+    """
+    # Get distance of planet at `time`
+    earth_to_planet_vector, earth_distance = _get_absolute_planet_position(time, planet_index)
+
+    # The apparent position depends on the time that the light was emitted from
+    # the distant planet, so subtract off the light travel time
+    light_travel_time = earth_distance/speed_of_light
+    emitted_time = time - light_travel_time
+
+    # Calculate position given approximate light travel time.
+    # TODO: this should be solved iteratively to converge on precise positions
+    earth_to_planet_vector, earth_distance = _get_absolute_planet_position(emitted_time, planet_index)
+    x, y, z = earth_to_planet_vector
+
+    return CartesianRepresentation(x=x, y=y, z=z)
+
+
+def get_planet(time, planet_index, location=None):
+    """
+    Get a `~astropy.coordinates.SkyCoord` for a planet as observed from a
+    location on Earth.
+
+    Parameters
+    ----------
+    time : `~astropy.time.Time`
+        Time of observation
+
+    planet_index : int
```

Comment:
I don't really see that as a half-way compromise in terms of ease of use, because that would require 20 characters to get mercury instead of one, and anyone needing multiple planets would need to import a bunch of extra variables to use the tools.